main.js

```diff
@@ -1 +1,51 @@
 // write your code here to make the tests pass
+
+var Library = function () {
+  var books = [];
+  var addBook = function(someBookObj) {
+    books.push(someBookObj);
+  };
+  var checkOutBook = function(bookObj) {
+    for (var i = 0; i < books.length; i++){
+      if (books[i] === bookObj) {
+        bookObj.setAttribute("checkedOut", true);
+      }
+    }
+  };
+  var returnBook = function(bookObj){
```

Comment:
Because the `addBook`, `checkOutBook` and `returnBook` are all accepting the same book object as arguments, perhaps naming the parameter the same name across all of the functions might be more clear